onig_sys/build.rs

```diff
@@ -189,10 +189,18 @@ fn compile() {
 fn bindgen_headers(_path: &str) {
     #[cfg(feature = "generate")]
     {
+        let arch = env::var("CARGO_CFG_TARGET_ARCH").unwrap();
+        let os = env::var("CARGO_CFG_TARGET_OS").unwrap();
         let bindings = bindgen::Builder::default()
             .header(_path)
             .derive_eq(true)
             .layout_tests(false)
+            .clang_arg(
+            if arch == "aarch64" && os == "ios" {
```

Comment:
```suggestion
            // Work around for #rust-Lang/rust-bindgen/1211
            if arch == "aarch64" && os == "ios" {
```

Reply:
Looks like that issue got resolved.

Reply:
Do we need this PR any more? #147 Updates to bindgen .55. Hopefully that contains the fix.